apps/dev/src/commands/feeds/read/adfs.ts

```diff
@@ -9,9 +9,43 @@ import {
 import { renderTui, drawTable } from '@blocksense/base-utils/tty';
 import { listEvmNetworks, readEvmDeployment } from '@blocksense/config-types';
 import { AggregatedDataFeedStoreConsumer } from '@blocksense/contracts/viem';
+import { formatTimestamp } from '../../utils';
 
 const availableNetworks = await listEvmNetworks();
 
+export const parseADFSData = (hexData: string) => {
+  const cleanHex = hexData.startsWith('0x') ? hexData.slice(2) : hexData;
+  const valuePart = cleanHex.slice(0, 48);
+  const valueHex = '0x' + valuePart.padEnd(64, '0');
```

Comment:
Setting this padding is incorrect.

The value is only in the first 48 bytes. So adding this padding makes a different value when you convert it to `bigint`
```suggestion
  const valueHex = '0x'+ cleanHex.slice(0, 48);
```